src/VisualStudio/IntegrationTest/TestSetup/TestExtensionErrorHandler.cs

```diff
@@ -23,25 +22,9 @@ public TestExtensionErrorHandler()
 
         public void HandleError(object sender, Exception exception)
         {
-            if (exception is ArgumentException argumentException
-                && argumentException.Message.Contains("SnapshotPoint")
-                && argumentException.StackTrace.Contains("Microsoft.VisualStudio.Text.Editor.Implementation.WpfTextView.ValidateBufferPosition"))
+            if (exception.Message == "RemotePartyTerminated" && new System.Diagnostics.StackTrace().ToString().Contains("CodeLens") ||
+                exception.Message == "Cannot access a disposed object.\r\nObject name: 'CodeLensHubClient'.")
```

Comment:
do we really need the message to be that specific?

also, can we link this to a bug on the codelens people to fix whatever is going on?

Reply:
I'm going to file a tracking bug in a bit to clean up removing this once we're on a newer image since we expect things to resolve then.